src/main/java/sggw/wzim/czasnawypad/controller/AttractionRatingController.java

```diff
@@ -0,0 +1,63 @@
+package sggw.wzim.czasnawypad.controller;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import sggw.wzim.czasnawypad.db.dto.AttractionAverageRatingDTO;
+import sggw.wzim.czasnawypad.db.dto.AttractionRatingDTO;
+import sggw.wzim.czasnawypad.db.dto.CreateAttractionRatingDTO;
+import sggw.wzim.czasnawypad.db.entity.Attraction;
+import sggw.wzim.czasnawypad.db.entity.User;
+import sggw.wzim.czasnawypad.service.AttractionRatingService;
+
+@RestController
+@RequestMapping("/ratings")
+@RequiredArgsConstructor
+public class AttractionRatingController {
+
+    private final AttractionRatingService ratingService;
+
+    @GetMapping("/attraction/{attractionId}")
+    public ResponseEntity<?> getRatingsForAttraction(@PathVariable Integer attractionId) {
+        List<AttractionRatingDTO> attractionRatings = ratingService.getRatingsForAttraction(attractionId);
+        return ResponseEntity.ok(attractionRatings);
+    }
+    
+    @GetMapping
+    public ResponseEntity<?> getUserRatings(Integer userId) {
```

Comment:
- I don't think we should be using wildcards here as type for `ResponseEntity`.
- What is the purpose of this endpoint? Do we get other user's ratings or the current user whose executing the API?

Reply:
I know it should be only logged in user but for now I left it like this. If its gonna be the reason its not working with frontend, I will change it to id from token